File: js/romanNumerals.js
exports.toRoman = function(num) {
        let arab = num
        let answer = []
        let romansObj = {
          1000: 'M',
          500: 'D',
          100: 'C',
          50: 'L',
          10: 'X',
          5: 'V',
          1: 'I',
        }
        let keyArr = Object.keys(romansObj)
        keyArr.reverse()
      
        for (let k of keyArr) {
          if (k === arab) {
            answer.push(romansObj[k])
          }
          if ((arab % k !== 0) && (arab % k < arab)) {
            let counter = Math.floor(arab / k);
      
            for (i = counter; i > 0; i--) {
              answer.push(romansObj[k])
            }
            arab = arab % k
          }
      
          if ((arab < 5) && (arab > 0)){
            let counter = Math.floor(arab / k)
            for (i = counter; i > 0; i--) {
              answer.push(romansObj[k])
            }}
           
      
          }
       
          return answer.join('').toString();
        }

File: python/roman_numerals.py
import math
def to_roman(num):
    romans_dict = {'M': 1000, 'CM': 900,'D': 500,'CD':400,'C': 100,'XC': 90,'L': 50,'XL': 40,'X': 10,'IX': 9,'V': 5,'IV': 4,'I': 1}
    answer= []
    for roman in romans_dict:
        arab= math.floor(num/int(romans_dict[roman]))
        num-= arab * romans_dict[roman]
        answer.append(roman*arab)
    return "".join(answer)